WildFly_Preview_34.0.0.Final/jakarta-core-jdk21.adoc

```diff
@@ -0,0 +1,103 @@
+= TCK Results
+:ext-relative: {outfilesuffix}
+:imagesdir: ../images/
+:server-name: WildFly Preview
+:server-version: 34.0.0.Final
+
+image:splash_wildflylogo_small.png[WildFly, align="center"]
+
+
+== Compatibility Certification Information
+
+Date:  October 29, 2024
+
+* Product Name, Version: {server-name} {server-version}
```

Comment:
Looks like substitution didn't happen for server-name + server-version?

Reply:
If you click "View file" it will, https://github.com/wildfly/certifications/blob/141ec6d3fba7e539ad3432d26dadd08349124956/WildFly_Preview_34.0.0.Final/jakarta-core-jdk17.adoc